src/main/java/io/codelex/exceptions/Exercise2.java

```diff
@@ -0,0 +1,25 @@
+package io.codelex.exceptions;
+
+public class Exercise2 {
+
+    public static void main(String[] args) {
+        methodA(25, 0);
+
+    }
+
+    public static void methodA(int number, int div) throws ArithmeticException {
+        try {
+            System.out.println(number + " / " + div + " = " + number / div);
+        } catch (ArithmeticException e) {
```

Comment:
`After each try{} block put a catch{} block which catches the exception, prints a stack trace, and throws the exception object to its caller.`